packages/solidity-contracts/contracts/fuelchain/FuelMessagePortal/v3/FuelMessagePortalV3.sol

```diff
@@ -272,6 +272,6 @@ contract FuelMessagePortalV3 is FuelMessagePortalV2 {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[49] private __gap;
+    uint256[48] private __gap;
```

Comment:
we added 4 storage vars from the v2 version so should we set this accordingly ?

Reply:
Only `rateLimitEnabled` was added from current mainnet implementation at `0x6d67857224f66d7a677f063b861b6bacafb10639`